base/array.jl

```diff
@@ -1251,10 +1251,11 @@ julia> findnext(A,3)
 function findnext(A, start::Integer)
     for i = start:length(A)
         if A[i] != 0
-            return i
```

Comment:
They are, but sometimes you don't want to factor a single loop into a function just so that you can use a `return` to get a value out. This basically lets you have that without a function.